src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -213,6 +215,44 @@ protected final String getUriString(String filename) {
                 .toString();
     }
 
+    /**
+     * Returns the absolute system path of the given file.
+     *
+     * @param filename file name.
+     * @return absolute file path.
+     */
+    protected final File getInputFile(String filename) {
```

Comment:
Why can't we reuse `getPath` method instead of creating new one?

Reply:
You're right. There was a reason (might not be justified) why I did it this, way which I forgot. :)
Done.